server/tests/fixtures/model/userData.js

```diff
@@ -0,0 +1,16 @@
+import uuid4 from "uuid/v4";
```

Comment:
Please use https://www.npmjs.com/package/uniqid it is lighter and faster than uuid with less dependencies